aesm-client/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "aesm-client"
-version = "0.4.0"
+version = "0.4.1"
```

Comment:
Please undo this change, will conflict with #240

Reply:
reverted file